build/release/containers.sh

```diff
@@ -5,57 +5,76 @@ source ${scriptdir}/common.sh
 
 registry=${RELEASE_REGISTRY}/
 
-build() {
+get_image_name() {
     local os=$1
     local arch=$2
+    local repo=$3
+    local version=$4
 
-    [[ ${os} == "linux" ]] || return 0
+    local tag=${repo}-${arch}:${version}
+    if [[ ${arch} == "amd64" ]]; then
+        tag=${repo}:${version}
+    fi
 
-    tmpdir=$(mktemp -d)
-    trap "rm -fr $tmpdir" EXIT
-    cat <<EOF > $tmpdir/Dockerfile
-FROM alpine:3.5
-RUN apk add --no-cache gptfdisk util-linux coreutils
-COPY root /
-ENTRYPOINT ["/usr/bin/rookd"]
-EOF
-    build_artifact $os $arch $tmpdir rook/rookd rookd rook
-    rm -fr $tmpdir
+    echo ${tag}
 }
 
 build_artifact() {
     local os=$1
     local arch=$2
     local tmpdir=$3
     local repo=$4
-    local version=${RELEASE_VERSION}
 
     shift 4
     local bins="$@"
 
     layout_root $os $arch $tmpdir $bins
     mkdir $tmpdir/root/tmp
 
-    tag=${repo}-${arch}:${version}
-    if [[ ${arch} == "amd64" ]]; then
-        tag=${repo}:${version}
-    fi
+    local name=$(get_image_name $os $arch $repo ${RELEASE_VERSION})
 
-    echo building docker container ${tag}
-    docker build -t ${registry}${tag} $tmpdir
+    echo building docker container ${name}
+    docker build -t ${registry}${name} $tmpdir
 
-    local file=${tag/\//-}
+    local file=${name/\//-}
     local file=${file/:/-}
     local dockerout=${file}.docker
     echo ${file}
 
     echo generate ACIs from docker containers
-    (cd ${RELEASE_DIR} && docker save -o ${dockerout} ${registry}${tag})
+    (cd ${RELEASE_DIR} && docker save -o ${dockerout} ${registry}${name})
     (cd ${RELEASE_DIR} && docker2aci ${dockerout})
+}
+
+build() {
+    local os=$1
+    local arch=$2
 
+    [[ ${os} == "linux" ]] || return 0
+
+    tmpdir=$(mktemp -d)
+    trap "rm -fr $tmpdir" EXIT
+    cat <<EOF > $tmpdir/Dockerfile
+FROM alpine:3.5
+RUN apk add --no-cache gptfdisk util-linux coreutils
```

Comment:
note i did add a new package here in https://github.com/rook/rook/pull/529/files#diff-4df0eb05973b0172a2639e945b2c2ce3R18, the `e2fsprogs` package, so that would need to be added here as well (not sure if your upcoming rebase would catch that so FYI)